### ISSUE TITLE
Fix sorting of players when rating not present.

### ISSUE THREAD

**Issue by lakinwecker** [Lichess4545/heltour#281]
```
Traceback:

File "/home/lichess4545/web/www.lichess4545.com/env/lib/python3.6/site-packages/django/core/handlers/exception.py" in inner
 41.             response = get_response(request)

File "/home/lichess4545/web/www.lichess4545.com/env/lib/python3.6/site-packages/django/core/handlers/base.py" in _legacy_get_response
 249.             response = self._get_response(request)

File "/home/lichess4545/web/www.lichess4545.com/env/lib/python3.6/site-packages/django/core/handlers/base.py" in _get_response
 187.                 response = self.process_exception_by_middleware(e, request)

File "/home/lichess4545/web/www.lichess4545.com/env/lib/python3.6/site-packages/django/core/handlers/base.py" in _get_response
 185.                 response = wrapped_callback(request, *callback_args, **callback_kwargs)

File "/home/lichess4545/web/www.lichess4545.com/env/lib/python3.6/site-packages/django/utils/decorators.py" in _wrapped_view
 149.                     response = view_func(request, *args, **kwargs)

File "/home/lichess4545/web/www.lichess4545.com/env/lib/python3.6/site-packages/django/views/decorators/cache.py" in _wrapped_view_func
 57.         response = view_func(request, *args, **kwargs)

File "/home/lichess4545/web/www.lichess4545.com/env/lib/python3.6/site-packages/django/contrib/admin/sites.py" in inner
 224.             return view(request, *args, **kwargs)

File "/home/lichess4545/web/www.lichess4545.com/current/heltour/tournament/admin.py" in manage_players_view
 813.             return self.team_manage_players_view(request, object_id)

File "/home/lichess4545/web/www.lichess4545.com/current/heltour/tournament/admin.py" in team_manage_players_view
 948.         unassigned_players = list(sorted(season_players - team_players - alternate_players, key=lambda p: p.rating_for(league), reverse=True))

Exception Type: TypeError at /admin/tournament/season/137/manage_players/
Exception Value: '<' not supported between instances of 'NoneType' and 'int'
Request information:


```

**Comment by lakinwecker** [comment]
This was caused by manually added data.

**Comment by hopffgam** [comment]
this was fixed with #418 where it happened with data that was not manually added.
